proto/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@serverless/sdk-schema",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
```

Comment:
I don't really see the harm in not including both. That way if someone wants to use this in an `module` only project it will work without work arounds.

Reply:
What exactly is the use case? Note that this package serves only us, and is not expected to be used by community libraries.

For ease of maintenance I wouldn't bloat our config with things of no confirmed purpose.

Reply:
Why would this not be used by community libraries? If the community libraries need to use the generic Span/Trace payloads they will indeed need to use this to make compliant payloads.

We can remove it if you really want, just following what we do for all our internal projects.

Reply:
I think we agreed that we will specify just our tags, and not attempt to require proto schemas for those coming from users and third-party libraries.

And for that reason in SDK, I do not plan to expose on user any proto specifics. In basic form there'll be `Trace`, `Span` JS classes, which users will use to setup spans and add tags (without any protobuf handling implied on them)

Protobuf is just for our internal communication, that's how I see it now.

Reply:
What's the point to re-create JS classes if we already have generated classes? That is just reinventing the wheel for no reason. We will have a generic `Span` class that is generated. Why would we not let users create third party libraries that leverage it. 

After our discussion, ingest will only support protobuf payloads, why make an abstraction?

Reply:
I think we are talking about two different user personas here. One would be someone creating Spans using the SDK, which is what I think you are talking about, and one would be someone creating say a Mongodb instrumentation library. 

I was talking about the later in this case.

Reply:
> One would be someone creating Spans using the SDK,

Yes exactly, and goal it to provide to such user, simple straightforward interfaces. We should not require from users familiarity with protobuf or push on them its design quirks

> which is what I think you are talking about, and one would be someone creating say a Mongodb instrumentation library.

Do you envision in such case, that such library will be created together with protobuf schemas which we will adapt in our ingestion server (on backend side)?

My impression was that, it'll be handled in the same manner as spans created via SDK directly, and that it won't change the schema set we used on the backend side.

Nonetheless, it's a discussion we can re-visit later, once we have the first instrumentation working